db/migrate/20220713131925_add_index_on_task_name_and_status_to_runs.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddIndexOnTaskNameAndStatusToRuns < ActiveRecord::Migration[6.0]
+  def change
+    add_index(:maintenance_tasks_runs, [:task_name, :status],
```

Comment:
I was under the impression using an order on index was only necessary for multi-column indices needing to sort in different directions (ie if we were doing `task_name asc, status desc`). I'm not sure adding the order here is doing anything? 

It's also worth noting MySQL doesn't actually support ordering indices until 8.0: https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index
> Note: [MySQL](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/MySQL.html) only supports index order from 8.0.1 onwards (earlier versions accepted the syntax but ignored it).